core/Piwik.php

```diff
@@ -186,7 +186,7 @@ public static function getCurrentUserEmail()
      */
     public static function getContactEmailAddresses(): array
     {
-        $contactAddresses = trim(Config::getInstance()->General['contact_email_address']);
+        $contactAddresses = isset(Config::getInstance()->General['contact_email_address']) ? trim(Config::getInstance()->General['contact_email_address']) : "";
```

Comment:
What's the problem here? `contact_email_address` should be defined always, as it's defined in `global.ini.php`
And even if not this would be simpler `trim(Config::getInstance()->General['contact_email_address'] ?? '')`